contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,200 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+**Author: [Ronak Lakhotia](https://github.com/RonakLakhotia)**                                                              
+Reviewers: [Tan Heng Yeow](https://github.com/tanhengyeow), [Jacob Li PengCheng](https://github.com/jacoblipech)
+
+## What is Angular?
+
+Angular, according to the [official website](https://angular.io/docs), is described as follows:
+
+>Angular is a platform that makes it easy to build applications with the web. Angular combines declarative templates, dependency injection, end to end tooling, and integrated best practices to solve development challenges. Angular empowers developers to build applications that live on the web, mobile, or the desktop.
+
+Let us understand some key terms starting with what <i>declarative templates</i> mean.
+Consider the example of creating a simple web application as shown below.
+
+![Example](First_example.png)
+
+The application displays the details of a person, that is the id and the name. The view component (i.e., the part users see) is defined in Angular using templates. These templates are nothing but HTML files. Additionally, Angular extends the vocabulary of HTML by allowing you to create your own custom tags.
+Here is the template for the above example.
+  
+```html
+<h2>Hello World</h2>
+<div><span>id: 1</span></div>
+<div>
+  <label>name:
+    <input text="hero.name" placeholder="name"/>
+  </label>
+</div>
+```
+
+The templates used in Angular are <i>declarative</i> because they describe <i>what</i> the view should look like rather than giving instructions on <i>how</i> to construct the view. In the above example, we declare what we want displayed (i.e., name and id) and Angular renders the view.
```

Comment:
Why is there an `id: 1`? Is it used at all?

Reply:
Yup , in the screenshot there is an id and name for John Doe